ams/server/src/main/java/com/netease/arctic/server/DefaultOptimizingService.java

```diff
@@ -130,17 +130,18 @@ public void touch(String authToken) {
 
   @Override
   public OptimizingTask pollTask(String authToken, int threadId) {
+    LOG.debug("Optimizer {} threadId={} pooling task", authToken, threadId);
```

Comment:
We should pay attention to syntax errors in log printing.

The log message here can be `Optimizer {} (thread id {}) try polling task`

Reply:
Ok, I will modify it.